### DISCUSSION TITLE
WiFi and returning to having the Pi start it's AP mode WiFi, if it cannot connect to the configured SSID

### DISCUSSION THREAD

**Discussion by Astrobirder** [mcguirepr89/BirdNET-Pi#893] [Q&A]
Is there a reasonable way to make the BirdnetPi image AUTOMATICALLY revert to the initial image state if it fails to connect to the configured WiFi SSD after a few tries?  

I'm taking my Pi on a Roadtrip and will be using it at another location.  I think that spot has an AP available, but I don't know it's SSID or credentials right now.  

I read the thread with the trick of setting your phone up as the same SSID / password as your home network, so you can get into the Pi and change things--and that is indeed a cool trick.

I was just wondering if there was a reasonable way to have it revert back into AP mode (instead of STATION) after a few failed attempts.

Thanks in advance for your insights.

**Comment by spambake** [comment]
This autohotspot is what I use for a field/home BirdnetPi:
https://www.raspberryconnect.com/projects/65-raspberrypi-hotspot-accesspoints/183-raspberry-pi-automatic-hotspot-and-static-hotspot-installer
Easy to install and use.  I run the installer and use option 2.  To test the hotspot at home I use option 6 - reboot to go back to normal operation.
In the field you scan for wifi devices and log in to the hotspot then use a browser to http://10.0.0.5 to get to the birdnetPi home page.
